### ISSUE TITLE
Calendar creates itself twice when navigating with the keyboard

### ISSUE THREAD

**Issue by dariiing** [cryptpad/cryptpad#1250]
### Description

<img width="129" alt="image" src="https://github.com/cryptpad/cryptpad/assets/95225431/32b4efc5-339d-4c92-8780-07e56b55d0f2">

### Steps to reproduce

1. Go to calendar
2. Using only the keyboard, try to create a new calendar, it will create the same calendar twice

### Expected behavior

using only the keyboard should have the same behavior as working with a mouse.

### What opreating system are you using?

Microsoft Windows

### What browsers are you seeing the problem on?

Mozilla Firefox, Google Chrome

### Extensions

.

### Version

5.4.1

**Comment by yflory** [comment]
The commit was pushed to the `toolbar` branch which won't be included in the 5.5 release. I'm removing the milestone.
